### PR TITLE
WebUI: Dynamically choose Quit/Reboot button label

### PR DESCRIPTION
**Pull request by OndrejZobal**
If Installation_System.type = "Boot_ISO", "Reboot" will be displayed instead of "Quit" on the quit button and the quit confirmation modal.

### CODE REVIEW

File: ui/webui/src/components/AnacondaWizard.jsx [hunk 1]
```diff
@@ -104,6 +104,7 @@ export const AnacondaWizard = ({ onAddErrorNotification, toggleContextHelp, titl
             setStepNotification={setStepNotification}
             isInProgress={isInProgress}
             setIsInProgress={setIsInProgress}
+            isQuitReboot={conf["Installation System"].type == "BOOT_ISO"}
```
Comment (VladimirSlavik) [inline ui/webui/src/components/AnacondaWizard.jsx:107]: Is this something that can be depended on? I'd imagine this is something we want a D-Bus call, not a conf prop...

So maybe add a TODO?

Comment (OndrejZobal) [reply ui/webui/src/components/AnacondaWizard.jsx:107]: In an internal issue number `INSTALLER-3394` @M4rtinK suggested using the config directly and config was already used for something else.

Comment (M4rtinK) [reply ui/webui/src/components/AnacondaWizard.jsx:107]: I think that's the idea about the config file - it's always loaded & the values should not change, so we can avoid making a DBus API for everything.

**Review by M4rtinK — APPROVED** [review]
Looks good to me, thanks! :)

**Review by VladimirSlavik — APPROVED** [review]

**Comment by M4rtinK** [comment]
Is something blocking this PR or can it be merged now ? :)

**Pull request merged**